stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3821 @@
+{
```

Comment:
<p>If keeping the widget I'd maybe remove the markdown cell above it (and change "the above widget" to "the widget below"). It did work for me in jupyter notebook FYI though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Moved.